src/storage/v2/replication/replication_client.cpp

```diff
@@ -29,11 +29,18 @@ ReplicationClient::ReplicationClient(Storage *storage, const memgraph::replicati
                                      const memgraph::replication::ReplicationEpoch *epoch)
     : name_{config.name},
       rpc_context_{CreateClientContext(config)},
-      rpc_client_{io::network::Endpoint(config.ip_address, config.port), &rpc_context_},
+      rpc_client_{io::network::Endpoint(), &rpc_context_},
```

Comment:
Surely `io::network::Endpoint{io::network::Endpoint::needs_resolving, config.ip_address, config.port}` would be fine here.